index.js

```diff
@@ -960,13 +960,13 @@ class GithubScm extends Scm {
 
             try {
                 const files = await this.breaker.runCommand({
-                    action: 'listFiles',
-                    scopeType: 'pulls',
+                    scopeType: 'paginate',
+                    route: 'GET /repos/:owner/:repo/pulls/:number/files',
                     token,
                     params: { owner, repo, number }
```

Comment:
Not sure if I understand the implementation, it looks like the call is default to paginate https://octokit.github.io/rest.js/#api-Pulls-listFiles. 
We can try to increase `per_page` param to `100`. Or if we want to call ALL of them then we need to make more calls until there is no more data.

Reply:
The max number of results that github returns is 300. The `paginate` function does the pagination internally, so we don't have to deal with the pagination. https://github.com/octokit/rest.js#pagination

Reply:
According to the documentation, looks like octokit handles pagination with `paginate` and magically will return all the results: https://github.com/octokit/rest.js#pagination

Reply:
oh cool cool! :+1